tracer/tracer.mli

```diff
@@ -0,0 +1,30 @@
+type t
+
+type trace_layers =
+  { app : bool
+  ; lsp : bool
+  ; jsonrpc : bool
+  }
+
+val trace_layers :
+  ?app:unit -> ?lsp:unit -> ?jsonrpc:unit -> unit -> trace_layers
```

Comment:
I'd pass the flags as booleans.

Reply:
The function sets the layer to true if the argument with that name is passed. 

I can change the flags to booleans and make the function to take true/false into account, ie user could pass `~app:false` explicity if they don't want tracing.

Reply:
Yes, that's what I prefer.